Clients/GUI/ViewModels/SignUpViewModel.cs

```diff
@@ -26,10 +33,11 @@ public class SignUpViewModel
 
         [Compare(nameof(Password))]
         public string ConfirmPassword { get; set; }
-
+        public byte[] Avatar { get; set; }
+        public string AvatarExtension { get; set; }
         public SignUpViewModel()
         {
-            Birthday = DateTime.Today;
+            Birthday = DateTime.Today.AddYears(-18);
```

Comment:
Well may be if you decided to change this, change it to some constant date

Reply:
If we set a constant, then in a few years this date will not be relevant